runtime/v2/binary.go

```diff
@@ -63,7 +63,7 @@ type binary struct {
 	bundle                 *Bundle
 }
 
-func (b *binary) Start(ctx context.Context, opts *types.Any, onClose func()) (_ *shim, err error) {
```

Comment:
I think this is a case where such a pool can either ride the context or be global, it shouldn't appear in the function definition if it is not required for the implementation.

Reply:
yes, I think so, too
I'll go to fix it 
thx for review ! :)